Hush/Assets/Plugins/Line Of Sight/Scripts/LOSObjectHider.cs

```diff
@@ -39,15 +39,33 @@ private void LateUpdate()
             // TODO Do a fancy reveal/hide animation w/ shaders here
             if (overrideCulling)
             {
-                GetComponent<Renderer>().enabled = isRevealed;
+                UpdateVisibility(isRevealed);
             }
             else if (m_Culler.enabled)
             {
-                GetComponent<Renderer>().enabled = m_Culler.Visibile;
+                UpdateVisibility(m_Culler.Visibile);
             }
             else if (m_VisibilityInfo != null && m_VisibilityInfo.isActiveAndEnabled)
             {
-                GetComponent<Renderer>().enabled = m_VisibilityInfo.Visibile;
+                 UpdateVisibility(m_VisibilityInfo.Visibile);
+            }
+        }
+
+        private void UpdateVisibility(bool visible)
+        {
+            var parentRenderer = GetComponent<Renderer>();
+            
+            if (parentRenderer != null)
+                parentRenderer.enabled = visible;
+
+            var childrenRender = GetComponentsInChildren<Renderer>();
```

Comment:
Could we just get the renderers in `Start()`, store it, and then just check there instead of re-getting them every update? It doesn't seems like the renderer ever changes during runtime

Reply:
Yeah! Didn't wanna modify the Plugin file too much but that's worth a change! @redknight990 do you think you could do it?

Reply:
Ah didn't realize it was part of a plugin. If it's not our code, I don't care.